cleanWeather-vue/5.md

```diff
@@ -0,0 +1,83 @@
+## Lesson 5. Completing the UI
+
+The last thing we need to do is to display an image according to the current forecast. OpenWeatherMap API sends back an icon name with a forecast, so we can use that to build a nice UI.
+
+First, upload several images. Click the `app` folder name and click `+`. Add one folder called `images` and one called `fonts`. Download all the images ,here](https://github.com/jlooper/cleanweather/tree/master/app/images) to your local, then upload them to the Playground by clicking on the images folder and choosing 'upload resources'. You can add all the images at once. 
```

Comment:
Looks like a formatting issue near “,here”.

Reply:
Might be worth having an image with an arrow to show exactly where that + button is.